test/jdk/java/awt/print/PrinterJob/DrawStringMethods.java

```diff
@@ -21,44 +21,59 @@
  * questions.
  */
 
-/**
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.font.FontRenderContext;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+import java.text.AttributedCharacterIterator;
+import java.text.AttributedString;
+
+import jtreg.SkippedException;
+
+/*
  * @test
  * @bug 4185019
  * @key printer
  * @summary Confirm that all of the drawString methods on Graphics2D
  *          work for printer graphics objects.
+ * @library /test/lib /java/awt/regtesthelpers
+ * @build PassFailJFrame jtreg.SkippedException
  * @run main/manual DrawStringMethods
  */
+public class DrawStringMethods implements Printable {
 
-import java.awt.*;
-import java.text.*;
-import java.awt.font.*;
-import java.awt.print.*;
+    private static final String INSTRUCTIONS =
+            " Confirm that the methods are printed.\n" +
+            " For Graphics: drawString, drawString, drawChars, drawBytes\n" +
+            " For Graphics2D: drawString, drawString, drawGlyphVector";
```

Comment:
Expand the instructions to explain that a print job is automatically created an printed. The tester is expected to verify that the listed methods are present on the page.